contracts/core/SettingsManager.sol

```diff
@@ -122,6 +122,19 @@ contract SettingsManager is ISettingsManager, Ownable, Constants {
         }
     }
 
+    function addDelegatesToBlackList(address[] memory _delegates) external {
+        require(operators.getOperatorLevel(msg.sender) >= uint8(1), "Invalid operator");
+        for (uint256 i = 0; i < _delegates.length; ++i) {
+            EnumerableSet.add(banWalletList, _delegates[i]);
+        }
+    }
+    function removeDelegatesToBlackList(address[] memory _delegates) external {
```

Comment:
consider rename removeDelegatesToBlackList to removeDelegatesFromBlackList

Reply:
right